### PR TITLE
removes illegal internal module declarations

### PR DESCRIPTION
**Pull request by FlorianTopf**
@rokerkony this is what I have seen today when `tslint` "accidentally" run over the sources. We should not declare an internal (proprietary) module in the `d.ts` file. Instead we can define an external module and use imports where ever it's needed.
The reason is simple: The proprietary module declaration pollutes the gobal namespace with `rocketGridDatatable` property. Even if in this case it is not transformed to js code and only used by the compiler - we should not use internal modules anymore.

I will open an issue to introduce `tslint` directly here.

### CODE REVIEW

File: demo/app/usersPresentationService.ts [hunk 1]
```diff
@@ -8,15 +14,15 @@ export default class UserPresentationService extends BasePresentationService {
...
         super(paginationLimitPerPage);
 
         this.service = service;
     }
 
-    public getDefaultSorting (): rocketGridDatatable.IGetAllSortingParameter {
+    public getDefaultSorting (): IGetAllSortingParameter {
         return [
-            {
+            <ISortingParameter>{
```
Comment (rokerkony) [inline demo/app/usersPresentationService.ts]: not needed

File: package.json [hunk 2]
```diff
@@ -22,7 +22,7 @@
     "Konrad Cerny <konrad.cerny@rocket-internet.de>",
     "Florian Topf <florian.topf@rocket-internet.de>"
   ],
-  "license": "GPL",
+  "license": "GPL-3.0",
```
Comment (rokerkony) [inline package.json]: can you please change it also in bower?

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "rocket-grid-datatable",
-  "version": "1.0.0",
+  "version": "1.0.2",
```
Comment (rokerkony) [inline package.json]: This PR makes a BC - new version will be `2.0.0`

+ change of the version should be separated from change to different commit/PR.

File: package.json (hunk 2 above)
Comment (FlorianTopf) [inline package.json]: jepp.

File: package.json (hunk 3 above)
Comment (FlorianTopf) [inline package.json]: yes, I removed it from this commit.
